File: src/main/java/com/post_office/NO3Box.java
package com.post_office;

public class NO3Box extends Box{

    float length = 23;
    float width = 14;
    int height = 13;
    @Override
    boolean validate(float length, float width, int height) {
        if(length < this.length && width < this.width && height < this.height)
            return  true;
        return false;
    }
}

File: src/main/kotlin/post_office/NO3Box.kt
package post_office

import post_office.Box

class NO3Box(override var length: Float = 23f, override var width: Float = 14f, override var height: Int = 13) : Box() {
    override fun validate(length: Float, width: Float, height: Int): Boolean {
        return length < this.length && width < this.width && height < this.height
    }
}